lib/receptacle/method_delegation.rb

```diff
@@ -105,17 +90,29 @@ def __run_wrappers(method_cache, input_args, high_arity = false)
     # @param args input args of the repository method
     # @param high_arity [Boolean] if are intended for a higher arity method
     # @return processed method args by before wrappers
-    def __run_before_wrappers(wrappers, method_name, args, high_arity = false)
+    def __run_before_wrappers(wrappers, method_name, all_args)
       wrappers.each do |wrapper|
         next unless wrapper.respond_to?(method_name)
 
-        args = if high_arity
-          wrapper.public_send(method_name, *args)
+        if all_args[:args].empty? && all_args[:kwargs].empty?
+          wrapper.public_send(method_name)
```

Comment:
a test for this case should be added

Reply:
and shouldn't this be 
```suggestion
          all_args = wrapper.public_send(method_name)
```

not 100% sure I understand this method correctly, but prior to your changes it returned whatever the wrapper returned, but now it does in this case return the input it got as all_args?

Reply:
Yeah, this is the case where the method doesn't expect any arguments. In this case we don't need to forward what is returned. The gem didn't support this case before, I mentioned it here https://github.com/andreaseger/receptacle/pull/13#discussion_r906037445